tsconfig.json

```diff
@@ -2,30 +2,20 @@
 	"compilerOptions": {
 		"strict": true,
 		"lib": [
-			"es5",
-			"es6",
 			"dom",
-			"es2015.core",
-			"es2015.collection",
-			"es2015.generator",
-			"es2015.iterable",
-			"es2015.promise",
-			"es2015.proxy",
-			"es2015.reflect",
-			"es2015.symbol",
-			"es2015.symbol.wellknown",
-			"esnext.asynciterable"
+			"ES2020"
 		],
 		"outDir": "./build/",
 		"allowSyntheticDefaultImports": true,
-		"target": "es5",								/* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', 'ES2021', or 'ESNEXT'. */
```

Comment:
I think we were compiling to these old js versions because for a period we run the  crawler on the browser, which depending to its level of up-to-dateness may or may not have the latest js versions. But now that we are only running on the server it doesn't matter.